src/pages/docs/packaging-applications/build-servers/codefresh-pipelines.md

```diff
@@ -141,6 +141,43 @@ Octopus Deploy steps and examples are available from the [Codefresh Marketplace]
 
 Each step includes one or two examples to help with setting up a workflow. Basic examples include only required arguments, and complex examples include both required and optional arguments.
 
+## Deploy a tenanted release
+To deploy a tenanted release, use the **octopusdeploy/deploy-release-tenanted** step. Provide the details for your Octopus instance, and the tenants you want to deploy to. You will need to provide either tenants or tenant tags. To deploy an untenanted release, use the **octopusdeploy/deploy-release** step.
```

Comment:
Nice callout for untenanted releases. Could it be a link to that section of the doc?